src/main/java/com/mattstuhring/songr/HomeController.java

```diff
@@ -0,0 +1,44 @@
+package com.mattstuhring.songr;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class HomeController {
+
+    @GetMapping("/hello")
+    public String getRoot() {
+        System.out.println("Somebody got the root");
+        return "helloworld";
+    }
+
+    @GetMapping("/capitalize/{words}")
+    public String getCapitalizeWords(@PathVariable String words, Model m) {
+        m.addAttribute("words", capitalizeWords(words));
+        return "capitalize";
+    }
+
+    @GetMapping("/reverse")
+    public String getReversedWords(@RequestParam(required = false, defaultValue = "I like to program in Java") String words, Model m) {
+        m.addAttribute("reversed", reverseOrderOfWords(words));
+        return "reversed";
+    }
+
+    public static String capitalizeWords(String words) {
```

Comment:
Creating a method that just called `.toUpperCase()` seems like too much effort to me. Why add all this extra code for something that can be done on one line?